### PR TITLE
create 'quast' symlink to quast.py script in QUAST easyconfigs

### PR DESCRIPTION
**Pull request by boegel**
Apparently users expect QUAST to be usable via the `quast` command...

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2540.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ced200b38a37f73545fd73818b111ee7 for a full test report.

**Comment by boegel** [comment]
@migueldiascosta Looking into that, less trivial than it seems in this case (it affects the `%(namelower)s.py --test` sanity check command, because `pip` doesn't install the test data)

File: easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -36,16 +36,19 @@ exts_list = [
...
+    'files': ['bin/%(namelower)s', 'bin/%(namelower)s.py', 'bin/meta%(namelower)s.py'],
     'dirs': ['lib/python%(pyshortver)s/site-packages'],
 }
 
 sanity_check_commands = [
     "wget quast.sf.net/test_data.tar.gz && "
     "tar xzf test_data.tar.gz && "
     "%(namelower)s.py --test && "
-    "rm ./test_data.tar.gz ./test_data -rf"
+    "rm ./test_data.tar.gz ./test_data -rf",
```
Comment (akesandgren) [inline easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2018b-Python-2.7.15.eb]: Ehhh, options should be first not last.

File: easybuild/easyconfigs/q/QUAST/QUAST-4.6.3-foss-2018a-Python-3.6.4.eb [hunk 2]
```diff
@@ -41,7 +43,8 @@ sanity_check_commands = [
     "wget quast.sf.net/test_data.tar.gz && "
     "tar xzf test_data.tar.gz && "
     "%(namelower)s.py --test && "
-    "rm ./test_data.tar.gz ./test_data -rf"
+    "rm ./test_data.tar.gz ./test_data -rf",
```
Comment (akesandgren) [inline easybuild/easyconfigs/q/QUAST/QUAST-4.6.3-foss-2018a-Python-3.6.4.eb]: Options first.

File: easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2019a-Python-2.7.15.eb [hunk 3]
```diff
@@ -40,16 +40,19 @@ exts_list = [
...
+    'files': ['bin/%(namelower)s', 'bin/%(namelower)s.py', 'bin/meta%(namelower)s.py'],
     'dirs': ['lib/python%(pyshortver)s/site-packages'],
 }
 
 sanity_check_commands = [
     "wget quast.sf.net/test_data.tar.gz && "
     "tar xzf test_data.tar.gz && "
     "%(namelower)s.py --test && "
-    "rm ./test_data.tar.gz ./test_data -rf"
+    "rm ./test_data.tar.gz ./test_data -rf",
```
Comment (akesandgren) [inline easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2019a-Python-2.7.15.eb]: Options first

File: easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2018b-Python-2.7.15.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2018b-Python-2.7.15.eb]: fixed in 5ff1336

**Comment by boegel** [comment]
I've reverted the changes to the QUAST 4.x easyconfigs, it's too painful to get those working now, and there's little value in doing so (old toolchains, etc.)

This should be good to go now, @akesandgren's remarks have also been handled.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node2697.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/04254b5207aa8a8662f10b4407b36d09 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
`quast` symlink is already there in the most recent `QUAST` easyconfigs, no planning to pursue the effort of trying to also put it in place for older `QUAST` easyconfigs...

**Pull request closed without merging**